templates/cube/faq.html

```diff
@@ -0,0 +1,64 @@
+      <h2>Exams</h2>
+      <h3 id="exams-format">What is the format of the exams?</h3>
+      <p>CUBE exams are administered as interactive labs. You will be provided with a virtual Ubuntu terminal that runs in your web browser, and will be given three separate problems to solve using the terminal. You are able to cycle among the questions, and may approach them in any order you wish.</p>
+      <p>You will have a certain amount of time to complete all of the questions. However, the amount of time you allocate to each question is up to you.</p>
+      <h3 id="exam-result">When will I know the results of my exam?</h3>
+      <p>You will know the results of your exam not long after submitting it. Please allow up to 2 business days for your results to be processed. The results will be visible under your profile on the <a href="/cube/microcerts">Microcertifications</a> page.</p>
+      <h3 id="pass-exam">What happens if I pass my exam?</h3>
+      <p>If you pass your exam, you will be awarded the corresponding microcert, and you will be granted a microbadge that you can share online. The microbadge and the links to share it will appear under the appropriate module on the <a href="/cube/microcerts">Microcertifications</a> page.</p>
+      <h3 id="fail-exam">What happens if I fail my exam?</h3>
+      <p>If you fail the exam, a message will appear under the corresponding module on the <a href="/cube/microcerts">Microcertifications</a> page. You will not be awarded the microcert for that exam.</p>
+      <p>You can purchase another enrollment in order to take the exam again.</p>
+      <h2>Preparatory Materials</h2>
+      <h3 id="differences">What is the difference between the Study Labs and Study Materials?</h3>
+      <p>The <a href="/cube/study">CUBE Study Materials</a> are available free of charge, and offer a brief overview of some of the topics covered in the CUBE exams.</p>
+      <p>The CUBE Study Labs are available for a small fee. They provide access to virtual Ubuntu terminals and related exercises that simulate the experience of taking an exam.</p>
+      <h3 id="preparatory-materials">Will the preparatory materials train me in using Ubuntu?</h3>
+      <p>No. The Study Materials and Study Labs are meant to refresh your existing knowledge. They are not designed to turn an inexperienced Ubuntu user into a professional.</p>
+      <h3 id="preparatory-materials-coverage">Do the preparatory materials cover everything in the exams?</h3>
+      <p>No. The exams explore a wide range of topics related to their particular subject areas. The preparatory materials cover many of these topics, but not all of them.</p>
+      <p>To see what kinds of topics each microcert covers, consult the “Topics” column on the <a href="/cube/microcerts">Microcertifications</a> page.</p>
+      <h2>Badges</h2>
+      <h3 id="what-are-badges">What are badges?</h3 id="what-are-badges">
```

Comment:
Oops copy paste error? `id` added to the closing tag

Reply:
Fixed! 👍